packages/core/src/PasswordInput/PasswordInput.js

```diff
@@ -2,24 +2,24 @@ import React, { useState } from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import {
-  Visibility,
-  VisibilityOff,
   Check,
   Success,
   SuccessOutline,
+  Visibility,
+  VisibilityOff,
 } from 'pcln-icons'
 import {
+  Badge,
   Flex,
-  Text,
+  IconButton,
+  IconField,
   Input,
   Label,
-  IconField,
-  Badge,
   ProgressBar,
-  IconButton,
-} from '../../src'
+} from '..'
+import { Text } from '../Text'
```

Comment:
This was causing issues being imported as part of the index file imports which is weird but I didn't want to spend to much time debugging